Makefile

```diff
@@ -49,4 +49,4 @@ migrate: keyspace
 
 .PHONY: keyspace
 keyspace:
-	@toolkit keyspace --host=$(SCYLLA_NODES) --keyspace=$(SCYLLA_KEYSPACE) --replication-factor="1" $(if $(SCYLLA_USERNAME), --user=$(SCYLLA_USERNAME),) $(if $(SCYLLA_PASSWORD),--password=$(SCYLLA_PASSWORD),)
+	@toolkit keyspace --host=$(SCYLLA_NODES) --keyspace=$(SCYLLA_KEYSPACE) --replication-factor="1" $(if $(SCYLLA_USERNAME), --user=$(SCYLLA_USERNAME),) $(if $(SCYLLA_PASSWORD),--password=$(SCYLLA_PASSWORD),) -d
```

Comment:
Since it's just us working on the project for now, I don't see a problem.

But when we update the readme, we need to make it explicit that it's using the -d flag, which drops entire keyspace.

Also certificate the "dev" not run with the flag on production :chart_with_upwards_trend:

Reply:
Well, the project will be in develop phrase for a while and mostly me and @Daniel-Boll will be maintaining the Rust side. Doesn't look a bug/issue for me but a feature!